requirements.txt

```
@@ -7,7 +7,7 @@ compliance-checker==5.3.0
 gunicorn>=20.1.0
 cc-plugin-ncei
 cc-plugin-glider
-cc-plugin-ugrid
+cc-plugin-ugrid >=1.1.0
```

I left a space there, sorry.

```suggestion
cc-plugin-ugrid>=1.1.0
```